templates/emails/collective.virtualcard.assigned.hbs

```diff
@@ -0,0 +1,31 @@
+Subject: Virtual Card assigned to {{collective.name}}
+
+
+{{> header}}
+
+<center>
+  <h1>
+    {{host.name}} assigned a new Virtual Card to {{collective.name}}
+  </h1>
+</center>
+
+<p>
+  {{host.name}} assigned a new Virtual Card to your collective {{collective.name}}. This means that you also agreed with
+  the policy defined by {{host.name}} for using Virtual Cards.
+</p>
+<p>
+  <strong>Please, make sure to keep all expenses up-to-date, uploading the payment receipt to every charge you have on
```

Comment:
<strong>Please, make sure to keep all expenses up-to-date, uploading the payment receipt to every charge you have on this card.</strong> After each charge, you will receive an email with an invitation to submit the receipt of the purchase made. Your host may pause the card for future charges until you do.

Reply:
For more info, please go ahead and read the virtual cards FAQs {{LINK}}